src/go/rpk/pkg/cli/cmd/cluster/license/info.go

```diff
@@ -65,4 +70,8 @@ Expires:         %v days
 Version:         %v
 `
 	fmt.Printf(licenseFormat, p.Organization, p.Type, p.Expires, p.Version)
+	if p.Expires < 30 && p.Expires >= 0 {
+		fmt.Fprintln(os.Stderr, "warning: your license will expire soon")
+		return
```

Comment:
It's a bit odd to have these conditionals so far away from each other (one check for < 0 above, another check here). I think these should both be in one place above, or both be here (and don't emit any warning on expired for json, instead exit 1)

Reply:
I did it this way because I wanted to print a warning for "license expiring soon" only in `--format plain`. With your feedback I'll move this to:

1. If `--format json` exit 1 if the license is expired
2. if `--format plain` exit 1 if the license is expired and print warning.
    2a. if `--format plain` exit 0 and print a warning for "license expiring soon"